maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -424,8 +424,27 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         waitForAppToSettle()
     }
 
-    fun openLink(link: String) {
-        driver.openLink(link)
+    fun openLink(link: String, autoVerify: Boolean) {
+        driver.openLink(link, autoVerify)
+        waitForAppToSettle()
+    }
+
+    fun autoVerifyApp(appId: String?, autoVerifyApp: (text: String) -> Unit) {
+        if (driver is AndroidDriver) {
```

Comment:
Let's try to avoid any OS-specific checks at the top level, otherwise it will get out of hand very quickly. Instead, you can add a no-op implementation of `autoVerifyApp` to other drivers

Reply:
Done 💯  I have done following changes:

1. Change autoVerify method to private 
2. Used autoVerify flag passed in openLink method.

This will avoid adding explicit method for this in driver.